solutions/remotemonitoring/single-vm/docker-compose.java.yml

```diff
@@ -121,13 +121,15 @@ services:
       - PCS_STORAGEADAPTER_WEBSERVICE_URL=http://storageadapter:9022/v1
       - PCS_DEVICESIMULATION_WEBSERVICE_URL=http://devicesimulation:9003/v1
       - PCS_TELEMETRY_WEBSERVICE_URL=http://telemetry:9004/v1
+      - PCS_SOLUTION_TYPE
       - PCS_AZUREMAPS_KEY
       - PCS_AUTH_ISSUER
       - PCS_AUTH_AUDIENCE
       - PCS_AUTH_REQUIRED
       - PCS_CORS_WHITELIST
       - PCS_APPLICATION_SECRET
       - APPLICATION_SECRET
+      - PCS_SEED_TEMPLATE
```

Comment:
This will need to be added for standard deployment sku as well. If you choose the PCS_SEED_TEMPLATE value to be `default` that will need the above mentioned change. If you treat empty as default seeding process then it is not required.